incubator/external-dns/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels: {{ include "labels" . | indent 4 }}
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels: {{ include "labels" . | indent 8 }}
```

Comment:
Is there a 'standard' set of labels charts should apply? It makes sense if they are uniform to me.